experimental/dds/tree2/src/index.ts

```diff
@@ -91,6 +91,7 @@ export {
 	Adapters,
 	FieldAdapter,
 	TreeAdapter,
+	MapTree,
```

Comment:
We are exporting new types here at the package level, meaning that we intend for end-users of SharedTree to be writing FieldGenerators. Is that what you intend? I imagined that this feature was for our internal use, at least for the time being.

Reply:
It is still exported in the api.md, have we filed a work item to remove that?

Reply:
Hi Taylor, I chatted with Noah about it and decided to merge it in with TODO comments and create a work item for it. Just created one!